.github/workflows/codecov_ats.yml

```diff
@@ -0,0 +1,105 @@
+
+name: IGNORE ME codecov-ats
```

Comment:
Is the ignore me needed here? Is it just in case the workflow fails?

Reply:
Yeah... the dev-infra team expressed a certain reservation to having something show ❌  in the commit status if it's just a test thing. I understand and quite agree with that.
 
As a dev it's annoying to see the commit status fail, specially for something you don't necessarily know what is. So the very visible IGNORE ME is helpful in that scenario